itests/kit/node_opts.go

```diff
@@ -215,6 +218,19 @@ func MutateSealingConfig(mut func(sc *config.SealingConfig)) NodeOpt {
 		})))
 }
 
+func F3Enabled(bootstrapEpoch abi.ChainEpoch, blockDelay time.Duration, finality abi.ChainEpoch) NodeOpt {
```

Comment:
```suggestion
func WithF3Enabled(bootstrapEpoch abi.ChainEpoch, blockDelay time.Duration, finality abi.ChainEpoch) NodeOpt {
```

I know there's a mixed pattern in here but it'd be nice to default to `With` defaults